src/requirements/generator.ts

```diff
@@ -60,8 +62,17 @@ const produceSatisfiableCoursesAttachedRequirementJson = (): DecoratedRequiremen
         readonly requirements: readonly DecoratedCollegeOrMajorRequirement[];
       };
     };
+    minor: {
+      [key: string]: {
+        readonly name: string;
+        readonly schools: readonly string[];
+        readonly requirements: readonly DecoratedCollegeOrMajorRequirement[];
```

Comment:
Is this supposed to be "DecoratedCollegeOrMajorRequirement? If so, make an item to change the name of this

Reply:
Their structure is essentially the same in the reqs-function to the point that it's better to keep them consistent